pages/opengraph/collection/[collectionId].tsx

```diff
@@ -62,7 +62,7 @@ export default function OpenGraphCollectionPage() {
           />
         </div>
       </div>
-      <style jsx>{`
```

Comment:
issues w/ typescript. probs introduced because some weird peer deps issue. 

verified that this doesn't cause any issues.